benchmarks.sln

```diff
@@ -12,14 +12,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "experimental", "experimental", "{C2EB60CC-CA5A-4E49-A3F8-4432FB117869}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BasicAsyncHttpServer", "experimental\BasicAsyncHttpServer\BasicAsyncHttpServer.csproj", "{07773C38-B3F3-4D6C-B318-29C88F016AA9}"
```

Comment:
+1 and what's special about experimental\HttpBenchmark\HttpBenchmark.csproj that let's it live?